opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "metadata": {
+                "description": "Size of vm"
+            },
+            "defaultValue": "Standard_D2_v2"
+        },
+        "dnsLabelPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "adminUsername": {
+            "type": "string"
+        },
+        "adminPassword": {
+            "type": "secureString"
+        }
+    },
+    "variables": {
+        "vmName": "mongo-vm",
+        "nsgName": "mongo-nsg",
+        "nicName": "mongo-nic",
+        "publicIPAddressName": "mongo-publicip",
+        "publicIPAddressType": "Dynamic",
+        "virtualNetworkName": "net",
+        "vnetAddressPrefix": "10.0.0.0/16",
+        "subnetName": "mong-subnet",
+        "subnetPrefix": "10.0.0.0/24",
+        "dnsPrefix": "[concat(parameters('dnsLabelPrefix'), uniquestring(resourceGroup().id))]",
+        "mongoDBUser": "opencga",
+        "mongoDBPassword": "[concat('P', uniqueString(resourceGroup().id, '224F5A8B-51DB-46A3-A7C8-59B0DD584A41'), 'x', '!')]"
```

Comment:
So this might be me being way off here but I think this way of generating the password not mean that if anyone knows the `resourceGroup().id` then they can build the password. From memory the `UniqueString` method in ARM is more like a hashing method and as the template is in the open all the other inputs are also known. 

Maybe have missed something about how this one works, just being careful

Reply:
Parameterized username & password in https://github.com/opencb/opencga/pull/975/commits/1ec41d9d76237c8182b43e1b401c073730cfff12